dvb/cx23885/files/modules.txt

```diff
@@ -1,11 +1,188 @@
-modules.order
+modules.alias
```

Comment:
we don't need these, it'll be generated by depmod

Reply:
modinfo would disagree.

Reply:
I don't currently have, but is there any harm to include these?

Reply:
depmod will anyways re-generate that and it probably needs modules.order to do that right